app/script/calling/CallingRepository.js

```diff
@@ -482,6 +482,7 @@ z.calling.CallingRepository = class CallingRepository {
         if (response && isSelfUser) {
           const conversationName = callEntity.conversationEntity.display_name();
           this.callLogger.info(`Incoming call in conversation '${conversationName}' accepted on other device`);
+          this.mediaStreamHandler.releaseMediaStream();
           return this.deleteCall(conversationId);
```

Comment:
While this fixes the issue the root problem can be found in `needsMediaStream`.

Looking at above code you can see that `deleteCall()` is triggered which will in turn end up in `resetMediaStream()`. This only releases the MediaStream if it is not needed elsewhere.